RecoLocalTracker/SiPixelRecHits/src/PixelCPEGeneric.cc

```diff
@@ -113,26 +113,26 @@ PixelCPEGeneric::PixelCPEGeneric(edm::ParameterSet const & conf,
     xerr_endcap_def_=0.0020;
     yerr_endcap_= {0.00210};
     yerr_endcap_def_=0.00075;
-  } else { // isUpgrade=true, funny case
-    xerr_barrel_ln_= {0.00114,0.00104,0.00214};
-    xerr_barrel_ln_def_=0.00425;
-    yerr_barrel_ln_= {0.00299,0.00203,0.0023,0.00237,0.00233,0.00243,0.00232,0.00259,0.00176};
-    yerr_barrel_ln_def_=0.00245;
-    xerr_endcap_= {0.00151,0.000813,0.00221};
-    xerr_endcap_def_=0.00218;
-    yerr_endcap_= {0.00261,0.00107,0.00264};
-    yerr_endcap_def_=0.00357;
+  } else { // isUpgrade=true    
+    xerr_barrel_ln_= {0.00025, 0.00030, 0.00035, 0.00035};
```

Comment:
![c1_bpix_resx_nrows1_gauss](https://cloud.githubusercontent.com/assets/5084076/21096085/a44a8d22-c05e-11e6-85e0-5ae47bb035ed.png)

This is the plot you requested. The stdDev of the gaussian fit is indeed 2.1 um but conservatively I've assumed as error 2.5 um.